frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,8 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:5000/api',
+  authApiUrl: 'http://localhost:5000/api',
+  productApiUrl: 'http://localhost:3333/',
```

Comment:
You shoud have 1 apiUrl. You don't need to create 2 servers for each api points.
You need to create 1 server and use difference routes for getting data
http://localhost:5000/ - root route
http://localhost:5000/auth - auth route
http://localhost:5000/products - products route